Legion/ArtifactScenarios/Kruul.lua

```diff
@@ -85,7 +85,7 @@ function mod:OnBossEnable()
 	self:Log("SPELL_CAST_START", "DrainLife", 234423)
 	self:Log("SPELL_CAST_START", "HolyWard", 233473)
 	self:Log("SPELL_AURA_APPLIED_DOSE", "AuraOfDecay", 234422)
-	self:Log("SPELL_CAST_START", "Smash", 234631)
+	self:Log("SPELL_CAST_START", "Smash", 234631, 241717, 236537) -- Uses multiple
```

Comment:
in :Smash() you'll have to hardcode the id that matches the option `234631` instead of using `args.spellId` to allow the one option key defined in `:GetOptions()` to control the behavior for all 3 spell ids